Dockerfile

```diff
@@ -1,15 +1,18 @@
 FROM ruby:2.6.6-alpine as build-stage
 
 ENV RAILS_ENV=production
-ENV BUNDLE_APP_CONFIG="app/.bundle"
+ENV BUNDLE_WITHOUT="development:test"
+ENV BUNDLE_FROZEN="true"
 
 WORKDIR /app
-
-RUN apk add git build-base tzdata postgresql-dev
+RUN apk --update add git build-base postgresql-dev
 
 COPY . /app
 RUN gem update bundler --no-document
-RUN bundle install --without="development test"  --jobs 4 --retry 3
+RUN bundle install --jobs 4 --retry 3 \
+     && rm -rf /usr/local/bundle/cache/*.gem \
+     && find /usr/local/bundle/gems/ -name "*.c" -delete \
+     && find /usr/local/bundle/gems/ -name "*.o" -delete
```

Comment:
Neither the deprecated `--no-cache` nor it's ENV equivalent seem to work, not does setting the cache location to a disposable path such as `/tmp`.  see https://github.com/rubygems/rubygems/issues/3225

Using the typical workaround for now.